Makefile

```diff
@@ -1,24 +1,26 @@
+R := R
+
 .PHONY: all build check document test
 
 all: document build check
 
 build: document
-	R CMD build .
+	$(R) CMD build .
 
 check: build
-	R CMD check injectr*tar.gz
+	$(R) CMD check injectr*tar.gz
 
 clean:
 	-rm -f injectr*tar.gz
 	-rm -fr injectr.Rcheck
 	-rm -fr src/*.{o,so}
 
 document: clean
-	Rscript -e 'devtools::document()'
-	Rscript -e 'rmarkdown::render("README.Rmd")'
+	$(R) --no-echo -e 'devtools::document()'
```

Comment:
I have tried running with the --no-echo and it seems to also do the same thing? But this is where I do not have a ton of experience. What is the difference between running Rscript and an R interpreter with the args to run a command? Are there any differences?